tests/common/snappi_tests/snappi_test_params.py

```diff
@@ -48,3 +59,5 @@ def __init__(self):
         self.gen_background_traffic = True
         self.pause_flow_params = None
         self.poll_device_runtime = True
+        self.ecn_params = None
+        self.traffic_flow_config = TrafficFlowConfig()
```

Comment:
do you initialize pkt_size and pkt_count here?

Reply:
Yes it's a key in the traffic flow config type